src/Configuration/Profile.cs

```diff
@@ -527,31 +527,31 @@ public List<Gump> ReadGumps()
 
                         try
                         {
-                            GUMP_TYPE type = (GUMP_TYPE) int.Parse(xml.GetAttribute("type"));
-                            int x = int.Parse(xml.GetAttribute("x"));
-                            int y = int.Parse(xml.GetAttribute("y"));
-                            uint serial = uint.Parse(xml.GetAttribute("serial"));
+                            GumpType type = (GumpType) int.Parse(xml.GetAttribute(nameof(type)));
```

Comment:
Why did you prefer the nameof approach? Not because it’s wrong, i’m not a fan of floating strings. Im just curious

Reply:
Hmm looks like i forgot the other 3 hehe.

nameof() is just compiletime and the name is really the same, renaming stuff is easier for most IDEs - especially a VS without Resharper